### PR TITLE
fix parallel building

### PR DESCRIPTION
**Pull request by ThatNerdyPikachu**

### CODE REVIEW

**Comment by WinterMute** [comment]
Also needs https://github.com/switchbrew/switch-examples/blob/master/templates/application/Makefile#L99-L102 and https://github.com/switchbrew/switch-examples/blob/master/templates/application/Makefile#L179

**Comment by ThatNerdyPikachu** [comment]
Please tell me if I did it wrong, I suck at Makefiles

**Comment by WinterMute** [comment]
LGTM

File: Makefile.nx [hunk 1]
```diff
@@ -188,6 +190,8 @@ $(OUTPUT).elf	:	$(OFILES)
 	@echo $(notdir $<)
 	@$(bin2o)
 
+$(OFILES_SRC)	: $(HFILES_BIN)
```
Comment (yellows8) [inline Makefile.nx]: Should've been after the "$(OUTPUT).elf : $(OFILES)" line.

**Pull request merged**